html/ajax_table.php

```diff
@@ -25,7 +25,7 @@
 settype($current, 'integer');
 $rowCount = $_REQUEST['rowCount'];
 settype($rowCount, 'integer');
-if (isset($_REQUEST['sort']) && is_array($_POST['sort'])) {
+if (isset($_REQUEST['sort']) && is_array($_REQUEST['sort'])) {
```

Comment:
Guessing this was for testing?

Reply:
Yes and no. It uses REQUEST elsewhere so why not.

Reply:
just a thought, why not directly switch to filter_input on old code segments?
https://www.php.net/manual/en/function.filter-input.php

Reply:
Thats not a protection against sql injections

Reply:
it validates Types and escapes Strings ... thats what's the sanitize filte does
https://www.php.net/manual/en/filter.filters.sanitize.php